lib/client-assets.php

```diff
@@ -175,6 +175,13 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'build/core-data/index.js' ),
 		true
 	);
+	wp_register_script(
+		'wp-polyfill-domrect',
+		gutenberg_url( 'build/polyfills/DOMRect.js' ),
+		array(),
+		filemtime( gutenberg_dir_path() . 'build/polyfills/DOMRect.js' ),
```

Comment:
For consistency and to avoid potential issues with OS-specific treatment of capitalization in file names, I might suggest we call this file `dom-rect.js` instead.